client/cpp/kserve-api/samples/grpc_infer_dummy.cc

```diff
@@ -0,0 +1,182 @@
+        std::cerr << "error: received incorrect shapes for '" << name << "'"
+                  << std::endl;
+        exit(1);
+    }
+    std::string datatype;
+    FAIL_IF_ERR(
+        result->Datatype(name, &datatype),
+        "unable to get datatype for '" + name + "'");
+    // Validate datatype
+    if (datatype.compare("FP32") != 0) {
+        std::cerr << "error: received incorrect datatype for '" << name
+                  << "': " << datatype << std::endl;
+        exit(1);
+    }
+}
+
+}  // namespace
+
+int main(int argc, char** argv) {
+    cxxopts::Options opt("grpc_infer_dummy", "Sends requests via KServe gRPC API.");
+
+    // clang-format off
+    opt.add_options()
+    ("h,help", "Show this help message and exit")
+    ("grpc_address", "Specify url to grpc service. ", cxxopts::value<std::string>()->default_value("localhost"))
+    ("grpc_port", "Specify port to grpc service. ", cxxopts::value<std::string>()->default_value("9000"))
+    ("input_name", "Specify input tensor name. ", cxxopts::value<std::string>()->default_value("b"))
+    ("output_name", "Specify input tensor name. ", cxxopts::value<std::string>()->default_value("a"))
+    ("model_name", "Define model name, must be same as is in service. ", cxxopts::value<std::string>()->default_value("dummy"))
+    ("model_version", "Define model version.", cxxopts::value<std::string>())
+    ("timeout", "Request timeout.", cxxopts::value<int>()->default_value("0"))
+    ;
+    // clang-format on
+
+    auto args = opt.parse(argc, argv);
+
+    if (args.count("help")) {
+        std::cout << opt.help() << std::endl;
+        exit(0);
+    }
+
+    std::string input_name(args["input_name"].as<std::string>());
+    std::string output_name(args["output_name"].as<std::string>());
+
+    std::string url(args["grpc_address"].as<std::string>() + ":" + args["grpc_port"].as<std::string>());
+    std::string model_name = args["model_name"].as<std::string>();
+
+    // Create a InferenceServerGrpcClient instance to communicate with the
+    // server using gRPC protocol.
+    std::unique_ptr<tc::InferenceServerGrpcClient> client;
+
+    FAIL_IF_ERR(
+        tc::InferenceServerGrpcClient::Create(&client, url),
+        err);
+
+    std::vector<float> input_data(10);
+    for (size_t i = 0; i < 10; ++i) {
+        input_data[i] = i;
+    }
+
+    std::vector<int64_t> shape{1, 10};
+
+    // Initialize the inputs with the data.
+    tc::InferInput* input;
+
+    FAIL_IF_ERR(
+        tc::InferInput::Create(&input, input_name, shape, "FP32"),
+        "unable to get input");
+    std::shared_ptr<tc::InferInput> input_ptr;
+    input_ptr.reset(input);
+
+    FAIL_IF_ERR(
+        input_ptr->AppendRaw(
+            reinterpret_cast<uint8_t*>(&input_data[0]),
+            input_data.size() * sizeof(float)),
+        "unable to set data for input");
+
+    // Generate the outputs to be requested.
+    tc::InferRequestedOutput* output;
```

Comment:
I think, `outputs` parameter in KServe Inference API is optional and since model has only one output, we are interested in all of the outputs. Therefore maybe we can drop whole part with `outputs` ? Would it work?